.github/workflows/ci.yml

```diff
@@ -56,3 +59,100 @@ jobs:
             -
                 name: "Run friendsofphp/php-cs-fixer"
                 run: "vendor/bin/php-cs-fixer fix --dry-run --diff"
+
+    test:
+        name: "PHP ${{ matrix.php-version }} + Composer v2 + MAKER_TEST_VERSION: ${{ matrix.maker-test-version }}"
+
+        runs-on: ubuntu-latest
+
+        services:
+            mysql:
+                image: mysql:5.7
+                env:
+                    MYSQL_ROOT_PASSWORD: root
```

Comment:
could we use `MYSQL_ALLOW_EMPTY_PASSWORD: true` and use an empty password? That would allow phpunit.xml.dist to remain the same and (selfishly) make my life simpler locally ;)

Reply:
Should work, will try tomorrow 👍🏻